packages/atomic/cypress/integration/facet-utils.ts

```diff
@@ -52,7 +52,7 @@ export async function assertSortCriteria(
     '@coveoSearch',
     requestBodyOrder
   );
-  const firstRequestBodyFacets = requestBody.facets[0];
+  const firstRequestBodyFacets = (requestBody.facets as any)[0];
```

Comment:
As we talked in the channel this morning, I don't think we should continue to do tests that verify the request itself, but should focus on the end results we get (aka: facets are in the correct order).

In any case, why did you need to modify this here, with a cast to `any` ?

Reply:
We have 1 jira to review all the existing check `requestBody` later
I put as any, just to remove the red warning check for the code ,